libs/ledgerjs/packages/errors/src/helpers.ts

```diff
@@ -113,7 +113,7 @@ export const deserializeError = (object: any): Error => {
       }
     }
 
-    if (!error.stack && Error.captureStackTrace) {
+    if (error && !error.stack && Error.captureStackTrace) {
       Error.captureStackTrace(error, deserializeError);
     }
     return error;
```

Comment:
This can still return `undefined`, should maybe be reflected in the return type.

Reply:
fixed